docker-entrypoint.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e
+
+# print all comands to console if DEBUG is set
+if [[ ! -z "${DEBUG}" ]]; then
+    set -x
+fi
+NODE_ID_SEED=${NODE_ID_SEED:-$RANDOM}
```

Comment:
Why is this line actually needed?

Reply:
a node_id_seed always needs to be defined when starting a l2p node whether it is random or not, as based on that seed, the running node id will be generated